packages/plugin/vite/src/VitePlugin.ts

```diff
@@ -1,17 +1,22 @@
-import fs from 'node:fs/promises';
 import { AddressInfo } from 'node:net';
 import path from 'node:path';
 
 import { namedHookWithTaskFn, PluginBase } from '@electron-forge/plugin-base';
-import { ForgeMultiHookMap, StartResult } from '@electron-forge/shared-types';
+import { ForgeMultiHookMap, ResolvedForgeConfig, StartResult } from '@electron-forge/shared-types';
+import chalk from 'chalk';
 import debug from 'debug';
+import fs from 'fs-extra';
 // eslint-disable-next-line node/no-extraneous-import
 import { RollupWatcher } from 'rollup';
 import { default as vite } from 'vite';
 
 import { VitePluginConfig } from './Config';
+import { getFlatDependencies } from './util/package';
 import ViteConfigGenerator from './ViteConfig';
 
+// Convenient for user customization.
+export { resolveDependencies } from './util/package';
```

Comment:
question: can we document what this exported function is for?

Reply:
It does seem unreasonable. 👀